client/packages/system/src/Patient/CreatePatientModal/PatientResultsTab.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC, useEffect } from 'react';
+import {
+  BasicSpinner,
+  DataTable,
+  Typography,
+  useColumns,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { PatientPanel } from './PatientPanel';
+import { usePatient } from '../api';
+import { useCreatePatientStore } from '../hooks';
+import { PatientFragment } from '../api/operations.generated';
+
+export const PatientResultsTab: FC<PatientPanel> = ({ patient, value }) => {
+  const { mutateAsync, isLoading, data } = usePatient.utils.search();
+  const { updatePatient } = useCreatePatientStore();
+  const t = useTranslation('patients');
+
+  const columns = useColumns<PatientFragment>([
+    {
+      key: 'firstName',
+      label: 'label.first-name',
+    },
+    {
+      key: 'lastName',
+      label: 'label.last-name',
+    },
+    {
+      key: 'dateOfBirth',
+      label: 'label.date-of-birth',
+    },
+  ]);
+
+  useEffect(() => {
+    if (!isLoading && !!patient && !data && !!patient.canSearch) {
+      const { firstName, lastName } = patient;
+      mutateAsync({ firstName, lastName });
+    }
+  }, [patient, isLoading, data]);
+
+  useEffect(() => {
+    updatePatient({ canCreate: true });
```

Comment:
just restricting the OK button until the search is finished. perhaps not required 🤷